web/static/js/external.js

```diff
@@ -0,0 +1,42 @@
+import Net from './utilities/net'
```

Comment:
This is the new "unholy" object that contains (most) of the DOM manipulations and net requests.